### DISCUSSION TITLE
Third party npm package can't be imported (Vite 4)

### DISCUSSION THREAD

**Discussion by Dubzer** [vikejs/vike#585] [Discussion] [answered]
Hi! 
For some reason, I can't use npm package @fluentui/react-icons, with vue-plugin-ssr **and** Vite 4

(I don't really understand the message, since package.json in the example already has `"type": "module"`
![image](https://user-images.githubusercontent.com/18244287/210213779-80eb6fe1-a87b-43ae-8380-9a2ccace9a10.png)

Here's a few interesting things: 
1) if I change vite version to 3.2.5 and plugin-react to 2.2.0, everything works just fine.
2) If i use vite 4 without VPS, everything works too: https://stackblitz.com/edit/vitejs-vite-qwwptr?file=src/App.jsx

Reproduction: 
Take react-full example project, run `npm i @fluentui/react-icons`, add any icon component to the page (for example `AccessTime24Filled`), open the page in dev mode

To be honest, I'm kinda lost in this issue, but since it's only reproducible with VPS example project, I think it might be relevant 
Also, my node is v19.3.0

**Comment by brillout** [comment, marked as answer]
Have you seen https://vite-plugin-ssr.com/invalid-esm?

**Reply by Dubzer** [reply]
Thanks, It fixed an error! 
I'm surprised I haven't found it
